specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/stable/2020-03-01/inputs.json

```diff
@@ -0,0 +1,1084 @@
+      }
+    },
+    "AzureSqlReferenceInputDataSource": {
+      "description": "Describes an Azure SQL database reference input data source.",
+      "x-ms-discriminator-value": "Microsoft.Sql/Server/Database",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ReferenceInputDataSource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AzureSqlReferenceInputDataSourceProperties"
+        }
+      }
+    },
+    "AzureSqlReferenceInputDataSourceProperties": {
+      "properties": {
+        "server": {
+          "description": "This element is associated with the datasource element. This is the name of the server that contains the database that will be written to.",
+          "type": "string"
+        },
+        "database": {
+          "description": "This element is associated with the datasource element. This is the name of the database that output will be written to.",
+          "type": "string"
+        },
+        "user": {
+          "description": "This element is associated with the datasource element. This is the user name that will be used to connect to the SQL Database instance.",
+          "type": "string"
+        },
+        "password": {
+          "description": "This element is associated with the datasource element. This is the password that will be used to connect to the SQL Database instance.",
+          "type": "string"
+        },
+        "table": {
+          "description": "This element is associated with the datasource element. The name of the table in the Azure SQL database..",
+          "type": "string"
+        },
+        "refreshType": {
+          "description": "This element is associated with the datasource element. This element is of enum type. It indicates what kind of data refresh option do we want to use:Static/RefreshPeriodicallyWithFull/RefreshPeriodicallyWithDelta",
+          "type": "string"
+        },
+        "refreshRate": {
+          "description": "This element is associated with the datasource element. This indicates how frequently the data will be fetched from the database. It is of DateTime format.",
+          "type": "string"
+        },
+        "fullSnapshotQuery": {
+          "description": "This element is associated with the datasource element. This query is used to fetch data from the sql database.",
+          "type": "string"
+        },
+        "deltaSnapshotQuery": {
+          "description": "This element is associated with the datasource element. This query is used to fetch incremental changes from the SQL database. To use this option, we recommend using temporal tables in Azure SQL Database.",
+          "type": "string"
+        }
+      }
+    },
+    "EventSerializationType": {
+      "type": "string",
+      "description": "Indicates the type of serialization that the input or output uses. Required on PUT (CreateOrReplace) requests.",
+      "enum": [
+        "Csv",
+        "Avro",
+        "Json",
```

Comment:
Removing CustomClr from this enum is a new breaking change. Was it covered by your breaking change review/approval?